scripts/generate_can.py

```diff
@@ -54,7 +54,6 @@ def camel_to_snake_case(value):
         value = re.sub(r'[^a-zA-Z0-9]', '_', value)
         return value
 
```

Comment:
This file should encapsulate the logic in a function and call that function from a `if __name__ == '__main__'` block. as it stands rn, it just runs on module import and the code cannot be reused

Reply:
The new changes isn't what I meant. I meant for something like this:
```
def main():
  # logic


if __name__ == '__main__':
  main()
```

main can then just be all your original global logic (or better yet just calls other functions that split up the logic into high level blocks